cfg/config.yaml

```diff
@@ -37,6 +38,7 @@ master:
       - "hyperkube scheduler"
       - "hyperkube kube-scheduler"
       - "scheduler"
+      - "openshift start master controllers"
```

Comment:
I was looking at OKD 3.11 and I noticed `hypershift openshift-controller-manager` instead of the above.